src/site/content/en/blog/wakelock/index.md

```diff
@@ -63,6 +63,13 @@ Of course, there are plenty of other use cases:
 * A web-based presentation app that keeps the screen
   on during a presentation
 
+{% Aside 'success' %}
+  After implementing the Screen Wake Lock API,
+  *Betty Crocker*, a major cooking site in the US,
+  saw a 300% increase in purchase intent indicators for their users.
+  Read more about their impressive success in the 🍰&nbsp;[Betty Crocker case study](/betty-crocker/).
```

Comment:
```suggestion
  Read more in the 🍰&nbsp;[Betty Crocker case study](/betty-crocker/).
```
We should avoid subjective statements like "impressive successs" and rather just provide the facts and let the readers decide for themselves whether it's an impressive success. Otherwise this is great!

Reply:
To be quite honest this was just a filler word to make the link appear in its entirety on a new line.

Reply:
Now we have an ugly orphan:

<img width="824" alt="Screen Shot 2020-06-25 at 17 17 51" src="https://user-images.githubusercontent.com/145676/85747724-d128fd80-b707-11ea-8712-5182f5e3dc2e.png">